src/Synapse/Mapper/InserterTrait.php

```diff
@@ -58,7 +64,7 @@ protected function insertRow(AbstractEntity $entity, array $values)
 
         $result = $statement->execute();
 
-        if ($this->autoIncrementColumn && ! $values[$this->autoIncrementColumn]) {
+        if ($this->autoIncrementColumn && ! Arr::get($values, $this->autoIncrementColumn)) {
```

Comment:
Weren't we going to revert this change in lieu of the above Exception thrown in this file?